services/common/src/constants/assets.ts

```diff
@@ -1 +1,2 @@
 export { default as SMALL_PIN_SELECTED } from "../assets/images/small-pin-selected.svg";
```

Comment:
We have _several_ instances of the edit icon! I don't think we need another.
We've got VioletEditIcon on Core which I just diffchecked against EditIcon (on both) and the difference is only the fill colour (which it could pick based on a boolean param, or be set via CSS.).

I am also curious about the extra classes on the icons, which I haven't had to use- are they not rendering correctly? Or do we have specific CSS rules in those other cases that should be more broadly-applied?

Reply:
This has been removed, it now uses the same icons in the esup table. The extra classes used aren't needed when we keep the styling the same as esup table, its just if we were to stick to the styling of say the actions menu in the Permit table, without those extra classes the icon looks too small or it is missing the purple colour for Core and the blue for MineSpace.